l10n_cn_account_voucher/__openerp__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "l10n_cn_account_voucher",
+    "version": "9.0.1.0.0",
+    "category": "Account",
+    "website": "https://www.elico-corp.com",
+    "author": "Elico corp, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "account",
+        "l10n_cn_sequence_voucher",
```

Comment:
Hi, Try to review this pr, but can't find related depends 'l10n_cn_sequence_voucher', where can i download it?